### ISSUE TITLE
xsd:string example as identifier scheme

### ISSUE THREAD

**Issue by iherman** [w3c/epub-specs#1396]
[Example 83](https://w3c.github.io/epub-specs/epub33/core/#example-83) includes the following:

```xml
    <dc:identifier id="pub-id">urn:uuid:A1B0D67E-2E81-4DF5-9E67-A64CBE366809</dc:identifier>
    <meta refines="#pub-id" property="identifier-type" scheme="xsd:string">uuid</meta>
 ```

I presume the intention is that the `uuid` is some sort of a `xsd:string` derived type and serves as a constraint or indication on what the `pub-id` value is. However,

1. `uuid` is not a [primitive datatype](https://www.w3.org/TR/xmlschema-2/#built-in-datatypes) in XSD; `anyURI` is
2. Formally, `anyURI` is "just" another primitive datatype, that does not have any 'derived' relationship to `xsd:string`

I would expect what would work is something like:

```xml
    <dc:identifier id="pub-id">urn:uuid:A1B0D67E-2E81-4DF5-9E67-A64CBE366809</dc:identifier>
    <meta refines="#pub-id" property="identifier-type" scheme="https://www.w3.org/TR/xmlschema-2/">anyURI</meta>
 ```

**Comment by mattgarrish** [comment]
It's actually just a relic of an old [`datatype` attribute](http://idpf.org/epub/30/spec/epub30-publications-20110523.html#attrdef-meta-datatype) we floated in 3.0. That was needed before we went to CURIEs in a dedicated `scheme` attribute.

Looks like the attribute got incorrectly switched to `scheme` in that example instead of deleted. It's not even terribly useful, since the URN scheme already tells you the datatype more specifically.

I'd suggest we just delete it, as I believe your alternative is also invalid as the scheme attribute requires a CURIE.
